packages/pwa/app/utils/utils.js

```diff
@@ -159,12 +163,73 @@ export const capitalize = (text) => {
 
 /**
  * Get the pwa configuration object from pwa-kit.config.json
+ * @param path - path to your object inside pwaKitConfig using jsonPath expression
+ *
+ * To read more https://github.com/JSONPath-Plus/JSONPath
+ *
+ * @example getConfig('app.url') => {locale: 'path', site: 'path'}
  * @returns {object} - the configuration object
  */
-export const getConfig = () => pwaKitConfig
+export const getConfig = (path, opts) => {
+    if (!path) return pwaKitConfig
+    const result = JSONPath({path: `$.${path}`, json: pwaKitConfig, wrap: false, ...opts})
+    return result
+}
 
 /**
- * A util to return current url configuration
- * @returns {object} - url object from the pwa-kit.config.json file
+ * This function takes the url to determine the site, and then return its l10n configuration
+ * @param path
+ * @returns {object}
  */
-export const getUrlConfig = () => getConfig().url
+export const getL10nConfig = (path) => {
+    const sites = getConfig('app.sites.*')
+    if (!sites.length) throw new Error('No site config found. Please check you configuration')
+
+    const siteId = resolveSiteFromUrl(pathToUrl(path))?.id
+    const l10nConfig = sites.find((site) => site.id === siteId)?.l10n
+    return l10nConfig
+}
```

Comment:
You are mentioning `url` in the jsdoc summary, but the function is taking a path can you please rectify that?

I've also previously questioned the need for this function as it's simply a helper for the following and shouldn't be writtin anymore than this:

```
const getL10nConfig = (url) => {
    return resolveSiteFromUrl(url).l10n
}
```

Reply:
That is a good point! I don't think we need to have this function at all